PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+Task Summary: **Write in this line**
```

Comment:
❗ 
Task Summary 라는 글자도 필요없을것같은데요, 태스크 제시한대로 모든 커밋메세지와 PR에 불필요한 프리픽스가 남게됩니다.

다음과 같은 스타일은 어떤가요?  

```
[Abc]: Message.

e.g.
Add: New function to rule the world.  
Mod: Add women factor in Domination.ruleTheWorld().  
Ref: Extract empathy stuff to an abstract class.  
Fix: RUL-42 or #42 Starvation need to be initialised before Energy to avoid the nullpointer in People.  
Rem: freeSpeech is not used anymore.  
Rea: Removed old TODO and extra space in header.  
```

Reply:
아 이해했습니다!

첫줄에 원래 깃 브랜치 제목을 넣으려고 했는데.. parameter 를 활용하려면 git action 이 필요하여 우선은 수동으로 입력하게끔 template 을 해놓고 action 쪽 작업하면서 develop 할 수 있도록 하겠습니다!